docs/verifiable-document/document-store.md

```diff
@@ -4,8 +4,50 @@ title: Deploying Document Store Smart Contract
 sidebar_label: Deploying Document Store
 ---
 
-Overview
+The document store is a smart contract on the Ethereum network that records the issuance and revocation status of OA documents. In this guide, we will deploy a document store smart contract on the Ethereum `ropsten` network which is a test network that does not require actual ethers for transactions.
 
-## Header
+## Pre-requisite
 
-TBD
+- Chrome browser
+- Metamask connected to `ropsten` network
```

Comment:
just Metamask ? there is a step to switch to ropsten

Do you want to add a link t the appendix ?

Reply:
you can ignore the second comment there is a link above =)

Reply:
Sorry, I dont get the first comment either...